src/Servers/IIS/IIS/test/IIS.FunctionalTests/IIS.FunctionalTests.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <TestGroupName>IIS.FunctionalTests</TestGroupName>
-    <DisableFastUpToDateCheck>True</DisableFastUpToDateCheck>
```

Comment:
A bunch of the IIS stuff seems to explicitly disable the FastUpToDateCheck. Any idea why @jkotalik?

Reply:
Context: https://github.com/aspnet/IISIntegration/pull/1186

We should verify that VS will rebuild for native changes now.

Reply:
Fixed by adding some things to `UpToDateCheckInput`.